cmd/kops/create.go

```diff
@@ -80,7 +79,7 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 	}
 
 	// Codecs provides access to encoding and decoding for the scheme
-	codecs := k8sapi.Codecs //serializer.NewCodecFactory(scheme)
+	codecs := kopsapi.Codecs //serializer.NewCodecFactory(scheme)
```

Comment:
Switching from k8sapi -> kopsapi is suspicious.  But I think kopsapi is right, so ...